Gibe.AbTest/AbTest.cs

```diff
@@ -9,40 +10,62 @@ public class AbTest : IAbTest
 		private readonly IAbTestingService _abTestingService;
 		private readonly IRandomNumber _randomNumber;
 
-		public AbTest(IAbTestingService abTestingService, IRandomNumber randomNumber)
+		public AbTest(IAbTestingService abTestingService,
+			IRandomNumber randomNumber)
 		{
 			_abTestingService = abTestingService;
 			_randomNumber = randomNumber;
 		}
 
-		public Variation AssignVariation()
+		public IEnumerable<Experiment> AllExperiments()
+		{
+			return _abTestingService.GetExperiments().Where(x => x.Enabled);
```

Comment:
Probably a good idea to bake this logic into a query in the repository layer; this becomes inefficient if we do not clear out old 'not enabled' experiments.